index.html

```diff
@@ -0,0 +1,153 @@
+            <header class="second-section-title">header</header>
+            <p>text</p>
+          </div>
+        </section>
+        <section class="third-section">
+          <h2>Heading</h2>
+          <div class="site-wrapper">
+            <header class="project-list-header">header</header>
+            <ul class="project-list">
+              <li class="project-list-item">item 1
+                <section class="feature">
+                  <h2 class="title-feature">header</h2>
+                  <p class="content-feature">text</p>
+                  <ul class="link-list">
+                    <li class="link-list-item">1</li>
+                    <li class="link-list-item">2</li>
+                    <li class="link-list-item">3</li>
+                  </ul>
+                </section>
+              </li>
+            <li class="project-list-item">item 2
+              <section class="feature">
+                <h2 class="title-feature">header</h2>
+                <p class="content-feature">text</p>
+                <ul class="link-list">
+                  <li class="link-list-item">1</li>
+                  <li class="link-list-item">2</li>
+                  <li class="link-list-item">3</li>
+                  <li class="link-list-item">4</li>
+                </ul>
+              </section>
+            </li>
+              
+            <li class="project-list-item">item 3
+              <section class="feature">
+                <h2 class="title-feature">header</h2>
+                <p class="content-feature">text</p>
+                <ul class="link-list">
+                  <li class="link-list-item">1</li>
+                  <li class="link-list-item">2</li>
+                  <li class="link-list-item">3</li>
+                  <li class="link-list-item">4</li>
+                </ul>
+              </section>
+            </li>
+            <li class="project-list-item">item 4
+              <section class="feature">
+                <h2 class="title-feature">header</h2>
+                <p class="content-feature">text</p>
+                <ul class="link-list">
+                  <li class="link-list-item">1</li>
+                  <li class="link-list-item">2</li>
+                  <li class="link-list-item">3</li>
+                  <li class="link-list-item">4</li>
+                </ul>
+              </section>
+            </li>
+          </ul> 
+          <h2 class="project-list-title">header</h2>
+          <p class="app-feature">text</p>
+          <ul class="app-list">
+            <li class="app-list-item">1</li>
+            <li class="app-list-item">2</li>
+            <li class="app-list-item">3</li>
+          </ul>
+          <p class="app-feature">text</p>
+          <ul class="app-list">
+            <li class="app-list-item">1</li>
+            <li class="app-list-item">2</li>
+            <li class="app-list-item">3</li>
+          </ul>
+        </div> 
+        </section>
```

Comment:
You need to check this part of the code more carefully to fix the indentation.

Reply:
Done.